package.json

```diff
@@ -162,7 +162,34 @@
       "testRule": true
     },
     "rules": {
-      "no-mixed-spaces-and-tabs": 0
+      "array-callback-return": "error",
+      "dot-notation": "error",
+      "func-name-matching": "error",
+      "guard-for-in": "error",
+      "no-else-return": [
+        "error",
+        {
+          "allowElseIf": false
+        }
+      ],
+      "no-implicit-coercion": "error",
+      "no-lonely-if": "error",
+      "no-mixed-spaces-and-tabs": "off",
+      "no-unneeded-ternary": "error",
+      "prefer-arrow-callback": "off",
+      "prefer-const": "error",
+      "prefer-destructuring": [
+        "off",
+        {
+          "object": true,
+          "array": false
+        }
+      ],
```

Comment:
Why add disabled rule?

Reply:
Because if not, it's enabled and I don't like using destructuring for arrays.

Reply:
Oh, you mean the whole rule. It's because I wanted to experiment with enabling or not.

Reply:
But I'm not sure if we want this enforced or not. I removed it for now.